lib/rails_multisite/connection_management.rb

```diff
@@ -246,10 +256,6 @@ def establish_connection(opts)
         end
       else
         handler = @default_connection_handler
-        if !@established_default
-          handler_establish_connection(handler, spec)
```

Comment:
There is no need to establish the connection here. ActiveRecord does it in one of the initializer.